airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -348,6 +348,8 @@ public interface Configs {
    */
   String getFeatureFlagClient();
 
+  int getFlushBufferFrequencyInMinutes();
```

Comment:
I think this also needs to go into https://github.com/airbytehq/airbyte-platform-internal/blob/master/oss/airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java 

@cpdeethree can you confirm/deny that? my understanding is that the Configs stuff is truly platform code, i.e. maybe OSS users want to set these vars manually - so these changes need to get mirrored into the platform repo

Reply:
so @cgardens has been digging deep in here, but my understanding is that what exists after charles made his flush is undecided and needs to be replicated out to both repositories. Charles, correct me if I am wrong on this understanding as I don't have specific context on this file or why you decided to leave it in the purge